### DISCUSSION TITLE
Exporting circle anotations as VOC or COCO

### DISCUSSION THREAD

**Discussion by yurikleb** [wkentaro/labelme#1120] [Ideas / Feature requests]
I find the "Circle" as a very useful annotation tool as I'm dealing with mostly round objects in my data.

Is it possible to export the circle annotations as VOC or COCO format dataset?

I saw those scripts:
https://github.com/wkentaro/labelme/blob/master/examples/instance_segmentation/labelme2voc.py
https://github.com/wkentaro/labelme/blob/master/examples/instance_segmentation/labelme2coco.py

Which seem to convert Instance and Semantic segmentations to VOC and COCO.

Do those script work with circle annotations?

here is how one of those labels looks in the JSON created by labelme:
```   
 {
      "label": "cap",
      "line_color": null,
      "fill_color": null,
      "points": [
        [
          129,
          161
        ],
        [
          141,
          172
        ]
      ],
      "shape_type": "circle"
 }
```

**Comment by yurikleb** [comment]
Hey, after exchanging a few emails with the team at roboflow.com they are now supporting the LabelMe JSON format, which makes it really easy to convert to any other format: https://roboflow.com/formats/labelme-json
But still, need to test how it works with cirle tags.

**Comment by ryouchinsa** [comment]
RectLabel is an offline image annotation tool for object detection and segmentation.
Although this is not an open source program, with RectLabel, you can import the Labelme JSON files which include the circle object and export to the COCO JSON format.
https://rectlabel.com/help#labelme_to_xml

![スクリーンショット 2022-12-29 21 14 22](https://user-images.githubusercontent.com/1954306/209949955-18fe16ab-1bff-4f20-a75e-ed61bd10772e.png)

**Reply by yurikleb** [reply]
Thank you! Will try!
